python/ray/ml/examples/rllib_example.py

```diff
@@ -0,0 +1,88 @@
+import argparse
+
+import ray
+from ray.ml.config import RunConfig
+from ray.ml.train.integrations.rllib.rl_trainer import RLTrainer
+from ray.ml.result import Result
+from ray.rllib.agents.marwil import BCTrainer
+
+
+def train_rllib_bc_offline(
+    path: str, num_workers: int, use_gpu: bool = False
+) -> Result:
+    dataset = ray.data.read_json(
+        path, parallelism=num_workers, ray_remote_args={"num_cpus": 1}
+    )
+
+    trainer = RLTrainer(
+        run_config=RunConfig(stop={"training_iteration": 5}),
+        scaling_config={
+            "num_workers": num_workers,
+            "use_gpu": use_gpu,
+        },
+        datasets={"train": dataset},
+        algorithm=BCTrainer,
+        param_space={
+            "env": "CartPole-v0",
+            "framework": "tf",
+            "evaluation_num_workers": 1,
+            "evaluation_interval": 1,
+            "evaluation_config": {"input": "sampler"},
+        },
+    )
+    result = trainer.fit()
+    print(result.metrics)
+
+    return result
+
+
+def train_rllib_bc_online(num_workers: int, use_gpu: bool = False) -> Result:
```

Comment:
this should be train_rllib_ppo_online() now that you are training a PPO agent for cartopole env.

Reply:
Good catch, updated!